benchmarks/driver.sh

```diff
@@ -8,10 +8,10 @@ FISH_PATH=$1
 BENCHMARKS_DIR=$(dirname "$0")/benchmarks
 
 for benchmark in "$BENCHMARKS_DIR"/*; do
-    echo $(basename "$benchmark")
+    basename "$benchmark"
     ${FISH_PATH} --print-rusage-self $benchmark > /dev/null
     if command -v hyperfine >/dev/null 2>&1; then
-        hyperfine "${FISH_PATH} $benchmark > /dev/null"
+        hyperfine "${FISH_PATH} "$benchmark" > /dev/null"
```

Comment:
That's a NAK. This line is plain *broken* with spaces. Did this come out of shellcheck?

What needs to happen here is `hyperfine` needs to get "/usr/bin/fish /path/to/benchmark >/dev/null" as one argument, as if it passed it to `/bin/sh -c` (which I'm pretty sure is exactly what happens. This opens the quotes again so $benchmark is *un*quoted?

Reply:
Yes it did. I’ll revert.

Reply:
Actually, I see the problem now. Pushed a fix.